frontend/src/tests/pages/HomePage.test.js

```diff
@@ -47,7 +47,7 @@ describe("HomePage tests", () => {
         const title = screen.getByTestId("homePage-title");
         expect(title).toBeInTheDocument();
         expect(typeof (title.textContent)).toBe('string');
-        expect(title.textContent).toEqual('Howdy Farmer');
+        expect(title.textContent).toEqual('Howdy Farmer ');
```

Comment:
I don't understand why you would expect the text to not contain Phillip if the user fixtures contain `Phillip` here and you have mocked the `currentUser` endpoint.

Can you explain?   How is this test different from the one below?

Reply:
Fixed and refactored in [1a7d20a](https://github.com/ucsb-cs156-m23/proj-happycows-m23-10am-4/pull/67/commits/1a7d20a906098224b66ecd783ad0d4aaa722f09e). So the original reason was that for some reason, regardless of the test, the first render of the page in the test suite will not show the name. However, the subsequent tests will. This, in theory, should not happen, but it does. I'm personally not familiar enough with react to know why and I can't find a reason online. I left it blank as a temporary dev fix, and I forgot to go back and address it. Currently, the workaround is to utilize the `expected CSS properties` test as the first test of the suite, as it does not check for the name. I'll address it properly with my team on Tuesday and any additional guidance is appreciated. Thanks!

Reply:
await waitFor was needed to test render correctly. Running local mutation tests now to see if anything else is needed

Reply:
Thank you so much!